src/imageio/format/jpeg.c

```diff
@@ -49,11 +49,13 @@ typedef struct dt_imageio_jpeg_t
   struct jpeg_decompress_struct dinfo;
   struct jpeg_compress_struct cinfo;
   FILE *f;
+  int subsample;
```

Comment:
Instead of the int here, can we have an enum `dt_jpeg_subsample_t` (or a better name) with all the possible values?

```
DT_SUBSAMPLE_AUTO
DT_SUBSAMPLE_444
...
```

and ...

Reply:
OK, converted to enum.